phpstan.neon

```diff
@@ -0,0 +1,18 @@
+parameters:
+#    level: 8
```

Comment:
Wouldn't it be good if the level was defined in phpstan.neon instead of in the composer.json command? So the configuration would be in one place.

```suggestion
    level: 8
```